src/tools/run-make-support/src/assertion_helpers.rs

```diff
@@ -0,0 +1,142 @@
+//! Collection of assertions and assertion-related helpers.
+
+use std::path::{Path, PathBuf};
+use std::panic;
+
+use crate::fs_wrapper;
+use crate::path_helpers::cwd;
+
+/// Browse the directory `path` non-recursively and return all files which respect the parameters
+/// outlined by `closure`.
+#[track_caller]
+pub fn shallow_find_files<P: AsRef<Path>, F: Fn(&PathBuf) -> bool>(
```

Comment:
I would put this under a `path` helper, or a `fs` helper.

Reply:
Moved to `path_heleprs`.